app/javascript/moderationArticles/singleArticle/index.jsx

```diff
@@ -17,6 +17,42 @@ const SingleArticle = ({
     );
   });
 
+  const months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+
+  const get12HourTime = (date) => {
+    const minutes = date.getMinutes();
+    let hours = date.getHours();
+    const AmOrPm = hours >= 12 ? 'PM' : 'AM';
+    hours = hours % 12 || 12;
+
+    return `${hours}:${minutes} ${AmOrPm}`;
+  };
+
+  const formatDate = (timestamp) => {
```

Comment:
I believe that what this `formatDate` function does can be done more elegantly in the `moderators` controller. I need some help with that though. I tried using the `article.rb` method called `readable_publish_date` but it wouldn't render on the page. So I did things this way just so I could generate the UI image.